lib/utils.js

```diff
@@ -94,6 +96,21 @@ exports.utils = {
         emit(exports, 'intent', 'destroy');
     },
     /**
+    * Calculates the amount of time remaining until the next
+    * notification timer
+    * @returns The next CTA complete time
+    */
+    getNextTimerCompleteTime: function() {
+        let lastCTATime = storageManager.get('lastSidebarCTACompleteTime');
```

Comment:
Would this be more easily testable if you passed this value in as a parameter? Also are you planning to add a test here in this PR?